config/jobs/kubernetes/minikube/minikube.yaml

```diff
@@ -15,6 +15,23 @@ presets:
 
 presubmits:
   kubernetes/minikube:
+  - name: pull-minikube-build
+    decorate: true
+    path_alias: "k8s.io/minikube"
+    always_run: true
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-minikube/prow-test:v0.0.1
```

Comment:
Where is the source for this image? How can we independently reproduce this image? Let alone ensure there's not a Bitcoin miner tucked inside (not suggesting you would do this, just giving a colorful example). I consider this a blocker. 

And again, what does this image have that is missing from the existing set of images available at gcr.io/k8s-testimages? Could we use one of those instead? I don't consider this a blocker.

Reply:
1. This is the image: https://github.com/kubernetes/minikube/tree/master/deploy/prow. If you want to build it. Just need to download minikube git repo and make push-prow-test-image. 
2. I haven't looked images gcr.io/k8s-testimages, There might be one i could use. The reason why we keep our own images for minikube is that we want to have everything managed on our own instead of have some dependency from some other images.

Reply:
SGTM, thanks for the answers.

it might be worth considering an image-pushing job and moving the image to community-hosted infrastructure at some point (ref: https://github.com/kubernetes/test-infra/tree/master/config/jobs/image-pushing#image-pushing-jobs)

Reply:
Sure, will work on that for the next pr, Thanks for the info!

Reply:
Now it's merged, will the prow job run on every minikube pr? Or do i need to configure something to make the prow job run on every minikube pr? Thanks!